tests/datasets/test_idtrees.py

```diff
@@ -140,15 +139,9 @@ def test_plot(self, dataset: IDTReeS) -> None:
             dataset.plot(x, show_titles=False)
             plt.close()
 
-    @pytest.mark.skipif(
-        sys.platform in ["darwin", "win32"],
-        reason="segmentation fault on macOS and windows",
-    )
     def test_plot_las(self, dataset: IDTReeS) -> None:
-        pytest.importorskip("open3d", minversion="0.11.2")
-        vis = dataset.plot_las(index=0, colormap="BrBG")
-        vis.close()
-        vis = dataset.plot_las(index=0, colormap=None)
-        vis.close()
-        vis = dataset.plot_las(index=1, colormap=None)
-        vis.close()
+        pyvista = pytest.importorskip("pyvista", minversion="0.35.1")
```

Comment:
@isaaccorley do you remember why our testing minversion is different from our setup.cfg min version? I tried correcting this in #1276 but it's failing. Is 0.35.1 the real min version that works? I don't want to claim to support versions we can't test.

Reply:
Might be an oversight on my part. I don't remember the reason exactly. I recall there was something to do with certain wheels for different OS not being available maybe?

Reply:
Wheels are pure Python, but could be limited by VTK. I'm going to try to find the oldest version for which our tests actually pass. We haven't been testing pyvista at all in our minimum version CI.

Reply:
>  but could be limited by VTK

Most likely the case

> I'm going to try to find the oldest version for which our tests actually pass.

I don't recommend this (see above comment https://github.com/microsoft/torchgeo/pull/663#discussion_r1043542639). Once you go back to older versions of PyVista (or let users do this), you'll need to limit VTK carefully. We work tirelessly to make sure the latest PyVista is compatible with all of VTK 9+, but older versions of PyVista may outright fail on import with newer versions of VTK. I recommend choosing a fairly recent version of PyVista as the minimum for this reason.

Reply:
Managed to get PyVista 0.29 working fine without having to mess with VTK: #1276